random.c

```diff
@@ -1296,6 +1296,21 @@ random_s_bytes(VALUE obj, VALUE len)
     return rand_bytes(&random_mt_if, rnd, NUM2LONG(rb_to_int(len)));
 }
 
+/*
+ * call-seq: Random.seed -> integer
+ *
+ * Returns the seed value used to initialize the generator. This may be used to
+ * initialize another generator with the same state at a later time, causing it
+ * to produce the same sequence of numbers.
```

Comment:
Feels “the generator” vague.

```suggestion
 * Returns the seed value used to initialize the Ruby system PRNG.
 * This may be used to initialize another generator with the same
 * state at a later time, causing it to produce the same sequence of
 * numbers.
```